modules/kolide-launcher/default.nix

```
@@ -87,6 +87,14 @@ in
       '';
     };
 
+    osQueryFlags = mkOption {
+      type = types.listOf types.str;
+      default = [];
+      description = ''
+        OsQuery flags to pass via the agent.
```

```suggestion
        Flags to pass to osquery (possibly overriding Launcher defaults).
```

I've been keeping the description for these flags consistent with how they're defined in launcher -- https://github.com/kolide/launcher/blob/main/pkg/launcher/options.go#L273